docs/mattermost-operator/mattermost-operator.yaml

```diff
@@ -4221,7 +4462,7 @@ spec:
           value: "20"
         - name: REQUEUE_ON_LIMIT_DELAY
           value: 20s
-        image: mattermost/mattermost-operator:v1.13.0
+        image: mattermost/mattermost-operator:latest
```

Comment:
@Szymongib is this correct?

Reply:
Yup, the version update will be handled with the direct commit that will be tagged as v1.14.0

Reply:
Yeah yeah, cool :)